app/Services/Campaign/ExportService.php

```diff
@@ -48,11 +53,25 @@ class ExportService
 
     protected int $currentElements;
 
+    protected EntityExportService $service;
```

Comment:
Naming a service `service` inside another service makes for difficult reading. Should have is called jsonEntityService` or `markdownEntityService`